packages/shared/friends/sagas.ts

```diff
@@ -2487,3 +2525,85 @@ function buildFriendRequestErrorResponse(error: FriendshipErrorCode) {
 function buildFriendRequestReply<T>(reply: NonNullable<T>) {
   return { reply, error: undefined }
 }
+
+/**
+ * Check whether the user has reached the max number allowed of sent requests to a given user.
+ * @param userId - the user to check the number of requests for.
+ * @returns true if the user has reached the max number of sent requests, false otherwise
+ */
+function reachedMaxNumberOfRequests(userId: string) {
+  // Get number friend requests sent in the current session to the given user
+  const sentRequests = getNumberOfFriendRequests(store.getState())
+  const number = sentRequests.get(userId) ?? 0
+
+  // Get the maximum number of requests allowed
+  const maxNumber = DEFAULT_MAX_NUMBER_OF_REQUESTS
+
+  // Check if the current number of requests is less than the maximum allowed
+  return number < maxNumber ?? false
```

Comment:
This seems to work opposite [to the use here](https://github.com/decentraland/kernel/pull/789/files#diff-bcea93a768b74b3a44000978a9f8e8f2b257395933ac042653b8153f0a1c751fR2220).

Also, the default expression `?? false` should not be necessary here as number and maxNumber must always be defined if I understand.

Reply:
Haha, right 🏃🏼‍♀️. I shouldn't have made this [commit](https://github.com/decentraland/kernel/pull/789/commits/664ea294bd1ab4e36c3198c1bb62eef270f89b74). It seems my brain got confused 🥴